winthrop/books/models.py

```diff
@@ -133,7 +146,75 @@ def add_creator(self, person, creator_type):
         Will throw an exception if creator type is not valid.'''
         creator_type = CreatorType.objects.get(name=creator_type)
         Creator.objects.create(person=person, creator_type=creator_type,
-            book=self)
+                               book=self)
+
```

Comment:
Should these have the `@staticmethod` decorator? Given how they're applied, I don't think it makes any difference (and perhaps the wrapper might cause issues?), but it might make it clearer that they're signal handlers associated with the model.